comp/trace/bundle_test.go

```diff
@@ -23,29 +24,25 @@ func TestBundleDependencies(t *testing.T) {
 	require.NoError(t, fx.ValidateApp(
 		// instantiate all of the core components, since this is not done
 		// automatically.
-		fx.Invoke(func(r config.Component) {}),
-		fx.Invoke(func(coreconfig.Component) {}),
```

Comment:
You should instanciate all your components to make the test useful.

Reply:
Fixed 😄 I finally understood `fx.Invoke`.